diagnostic.md

```diff
@@ -13,29 +13,25 @@ Record your responses inside the fenced code blocks below each question.
     `campus`?
 
     ```md
-    <!-- your response here -->
+    ember generate route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    {{#link-to 'campus' boston}} <h3> Random Link Name </h3> {{/link-to}}
     ```
 
 1.  Explain **at least** two differences between the following two route
     definitions.
 
     ```js
     this.route('products', function () {
-      this.route('product', { path: '/:product_id' }); // <= 👀here
+      this.route('product', { path: '/:product_id' }); // this is a route that is nested and the other one is not
```

Comment:
The first `'product'` route definition is nested under the `'products'`
definition, whereas the second `'product'` route definition is a sibling of
the `'products'` definition.

Nested route definitions, like the first `'product'` route above, require
a nested file structure, such that Route and Template files for `'product'`
are expected to be found at `app/products/product`. For sibling routes,
folders are siblings as well.

The first `'product'` route definition instructs Ember to render
`app/products/product/template.hbs` in the context (the `outlet`) of the
parent `app/products/template.hbs`. The second `'product'` route definition
instructs Ember to render `app/product/template.hbs` in the context of the
application template.

These two route definitions do **not** have different paths.

.5/1